### PR TITLE
EUI-5764 Multiday check your answers page

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5764

### Change description ###
Multiday- Check your answers page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/xui-webapp/Chart.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+    repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
   - name: redis
     version: 16.11.3
     repository: "https://charts.bitnami.com/bitnami"
     condition: redis.enabled
   - name: idam-pr
     version: ~2.2.6
-    repository: '@hmctspublic'
+    repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
     condition: idam-pr.enabled
```
Comment (rayliangatos) [inline charts/xui-webapp/Chart.yaml:19]: need to revert this

Comment (johnbenjamin-hmcts) [reply charts/xui-webapp/Chart.yaml:19]: @rayliangatos - I tried a few times yesterday, but it keeps coming back automatically during the build.

File: src/hearings/templates/hearing-view-edit-summary.template.ts [hunk 2]
```diff
@@ -213,7 +213,7 @@ export const HEARING_VIEW_EDIT_SUMMARY_TEMPLATE: Section[] = [
         id: 'hearingLength',
         answerTitle: 'Length of hearing',
         answerSource: AnswerSource.HEARING_LENGTH,
-        changeLink: '/hearings/request/hearing-timing#durationhours',
+        changeLink: '/hearings/request/hearing-timing#durationdays',
```
[the remark this answers is not in the document]
Comment (johnbenjamin-hmcts) [reply src/hearings/templates/hearing-view-edit-summary.template.ts:216]: Yes, we have now introduced Days for hearing length.

[the remark this answers is not in the document]
Comment (rayliangatos) [reply src/hearings/templates/hearing-view-edit-summary.template.ts:216]: ok seems that is in base branch

**Review by rayliangatos — APPROVED** [review]

**Review by Kartheek-S — APPROVED** [review]

**Pull request merged**